crochet/_util.py

```diff
@@ -15,3 +25,34 @@ def synced(self, *args, **kwargs):
             return method(self, *args, **kwargs)
     synced.synchronized = True
     return synced
+
+
+def get_twisted_reactor(use_global):
+    """
+    When you do `from twisted.internet import reactor`, twisted "installs" a
+    reactor. All this really means it that it looks at your platform to
+    determine which reactor (epoll, select, or poll) you should use, creates
+    a singleton of that type of reactor, and puts it in sys.modules as
+    `twisted.internet.reactor`.
+
+    This recreates logic in `twisted.internet.default` to choose between
+    reactors, but bypasses "installing" the reactor as a global singleton.
+    """
+    if use_global:
+        from twisted.internet import reactor
```

Comment:
This will actually install the reactor which is in contradiction with what the docstring says.

This might be more a problem with the docstring though. The docstring explains what use_global=False means rather than the function's complete behavior